featurebyte/service/offline_store_feature_table_comment.py

```diff
@@ -0,0 +1,172 @@
+
+        Returns
+        -------
+        TableComment
+        """
+        primary_entities = await self.entity_service.get_entities(
+            set(feature_table_model.primary_entity_ids)
+        )
+
+        def _format_entity(entity_model: EntityModel) -> str:
+            return f"{entity_model.name} (serving name: {entity_model.serving_names[0]})"
+
+        primary_entities_info = ", ".join([_format_entity(entity) for entity in primary_entities])
+        if feature_table_model.entity_lookup_info is not None:
+            parent_entity = await self.entity_service.get_document(
+                feature_table_model.entity_lookup_info.related_entity_id
+            )
+            sentences = [
+                f"This feature table is used to lookup the entity {_format_entity(parent_entity)}"
+                f" using the entity {primary_entities_info} based on their parent-child"
+                f" relationship"
+            ]
+        else:
+            sentences = [
+                f"This feature table consists of features for primary entity {primary_entities_info}"
+            ]
+        if feature_table_model.feature_job_setting:
+            job_setting = feature_table_model.feature_job_setting
+            sentences.append(
+                f"It is updated every {job_setting.frequency_seconds} second(s), with a blind spot"
+                f" of {job_setting.blind_spot_seconds} second(s) and a time modulo frequency of"
+                f" {job_setting.time_modulo_frequency_seconds} second(s)"
+            )
+        comment = ". ".join(sentences) + "."
+        return TableComment(table_name=feature_table_model.name, comment=comment)
+
+    async def generate_column_comments(
+        self, feature_models: List[FeatureModel]
+    ) -> List[ColumnComment]:
+        """
+        Generate comments for columns in offline feature tables corresponding to the features
+
+        Parameters
+        ----------
+        feature_models: List[FeatureModel]
+            Feature models
+
+        Returns
+        -------
+        List[ColumnComment]
+        """
+        # Mapping to from table name and column name to comments
+        comments: Dict[Tuple[str, str], str] = {}
+
+        for feature in feature_models:
+            feature_description = (
+                await self.feature_namespace_service.get_document(feature.feature_namespace_id)
+            ).description
+
+            offline_ingest_graphs = (
+                feature.offline_store_info.extract_offline_store_ingest_query_graphs()
+            )
+            for offline_ingest_graph in offline_ingest_graphs:
+                table_name = offline_ingest_graph.offline_store_table_name
+                if feature.offline_store_info.is_decomposed:
+                    comment = (
+                        f"This intermediate feature is used to compute the final feature"
+                        f" {feature.versioned_name}"
+                    )
```

Comment:
Would it be less confusing if we specify the feature name & version as `<feature_name> (version: <version>)?